api/src/main/java/io/strimzi/api/kafka/model/KafkaRebalanceSpec.java

```diff
@@ -33,6 +33,9 @@
     private List<String> goals;
     private boolean skipHardGoalCheck;
 
+    // Topic configuration
+    private String excludedTopicsRegex;
```

Comment:
I would use just "excludedTopics". We have documentation for explaining that it's a regex. The same applies to some other places where whitelist/blacklist is used (see Mirror Maker)

Reply:
Ok I will refactor it to `excludedTopics`